src/core/Tooltip/Tooltip.text.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { Tooltip } from './Tooltip';
+import { axeTest } from '../../utils/test';
+
+describe('props', () => {
```

Comment:
Further tests are needed to cover all necessary props.

Reply:
More tests are now added.